mac.sh

```diff
@@ -136,10 +136,10 @@ install_or_update_homebrew() {
   brew update
 }
 
-install_latest_ruby() {
-  append_to_zshrc 'eval "$(rbenv init - zsh --no-rehash)"' 1
+install_ruby_2_4_5() {
+  append_to_zshrc 'eval "$(rbenv init - --no-rehash zsh)"' 1
```

Comment:
Updating this based on [this thread](https://github.com/rbenv/rbenv/issues/856#issuecomment-170919066)

Reply:
thanks for the renaming here too